src/open_company_web/components/company_dashboard.cljs

```diff
@@ -42,6 +41,7 @@
      :columns-num (responsive/columns-num)})
 
   (did-mount [_]
+    (utils/after 100 #(set! (.-scrollTop (.-body js/document)) 0))
```

Comment:
comment here on why this is happening (from a user/UX perspective) would be good